manifests/forwarder.pp

```diff
@@ -171,7 +171,8 @@
     mode  => $facts['kernel'] ? {
       'windows' => undef,
       default   => '0600',
-    }
+    },
+    seluser => 'unconfined_u',
```

Comment:
I'm not sure this is right.  If on the second run, puppet wants to set it to `system_u` that's probably because a new selinux module has been installed with the RPM and the type **should** be `system_u`.  I think there's also a high probability users are bumping into https://tickets.puppetlabs.com/browse/PUP-2169 which `puppet apply` (as used in the acceptance tests) won't be affected by.

Basically, puppet defaults the value of `seluser` based on what it thinks it should be.  But, there's a caching issue and it never rechecks after a new selinux policy is loaded.  You need to restart the puppet service... :(